provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+docker build -t al2-graalvm .
```

Comment:
Should we use a different tag (something like `al2-graalvm-gradle` for this image, so that if someone tries both `maven` and `gradle` examples won't have any problems?